apis/r/R/SOMAExperimentAxisQuery.R

```diff
@@ -705,6 +705,259 @@ SOMAExperimentAxisQuery <- R6::R6Class(
       SeuratObject::DefaultAssay(mat) <- private$.measurement_name
       validObject(mat)
       return(mat)
+    },
+    #' @description Loads the query as a
+    #' \code{\link[SingleCellExperiment]{SingleCellExperiment}} object
+    #'
+    #' @param X_layers \Sexpr[results=rd]{tiledbsoma:::rd_outgest_xlayers('sce')}
+    #' @param obs_index \Sexpr[results=rd]{tiledbsoma:::rd_outgest_index('sce')}
+    #' @param var_index \Sexpr[results=rd]{tiledbsoma:::rd_outgest_index('sce', 'var')}
+    #' @param obsm_layers \Sexpr[results=rd]{tiledbsoma:::rd_outgest_mlayers('sce')}
+    #' @param obs_column_names \Sexpr[results=rd]{tiledbsoma:::rd_outgest_mdnames('sce')}
+    #' @param var_column_names \Sexpr[results=rd]{tiledbsoma:::rd_outgest_mdnames('sce', 'var')}
+    #' @param obsp_layers \Sexpr[results=rd]{tiledbsoma:::rd_outgest_players('sce')}
+    #' @param varp_layers \Sexpr[results=rd]{tiledbsoma:::rd_outgest_players('sce', 'varp')}
+    #'
+    #' @return A \code{\link[SingleCellExperiment]{SingleCellExperiment}} object
+    #'
+    to_single_cell_experiment = function(
+      X_layers = NULL,
+      obs_index = NULL,
+      var_index = NULL,
+      obs_column_names = NULL,
+      var_column_names = NULL,
+      obsm_layers = NULL,
+      obsp_layers = NULL,
+      varp_layers = NULL
```

Comment:
Is there _never_ `varm`??

Reply:
OK let's at least put a code comment saying it can't exist, and its omission is by thoughtful design.

We live in a polyglot world, and this is a polyglot project, and anyone with less R/Seurat expertise may be unnecessarily confused/concerned.

Reply:
@mojaveazure I maintain my ask

Reply:
There's a comment in the code highlighting the ommision of `varm_layers` from the parameter list (and has been since https://github.com/single-cell-data/TileDB-SOMA/pull/1279/commits/961aad3fca749b0a0b25f9d0b84beb240285a023)
![image](https://github.com/single-cell-data/TileDB-SOMA/assets/11035068/98bdfc6c-bb33-4845-914f-a9956d9ed0b8)